tutorials/examples/cov3d_simulation.py

```diff
@@ -52,6 +53,7 @@
 # Passing in a mean_kernel argument to the following constructor speeds up some calculations
 covar_estimator = CovarianceEstimator(sim, basis, mean_kernel=mean_estimator.kernel)
 covar_est = covar_estimator.estimate(mean_est, noise_variance)
+covar_est = covar_est.T # RCOPT
```

Comment:
I removed it and did not get acceptable `rel_err`.  I would like to look at this more and I will come back after I work through the remaining conversations.

Reply:
This is probably related to the estimation/kernel code...

Reply:
It must be. That's very concerning, though. We can leave it as a separate issue for now.

Reply:
Okay, so when I reshape to a 2D matrix, then we are symmetric, ie  `covar_est.reshape(8**3,8**3)`. I gather the transpose is more an artifact for the volume axes then. I should be able to clean this up.

Reply:
I've been looking at this Friday and this morning. I went to `develop` for comparison, and it is also not symmetric in the way I would have expected either, so this has probably been not quite right since it came into python. I think the conversion is tickling it, and that it is related to the outstanding estimation conversation. I see some related places yet to be converted so I'm working on those, hoping maybe can solve these two outstanding conversations together.

Reply:
> Okay, so when I reshape to a 2D matrix, then we are symmetric, ie covar_est.reshape(8**3,8**3).

Ok. I was under the impression that transposes and reshapes were commutative operations, but that is not correct. If I understand correctly, transposing the 2D matrix corresponds to `transpose((3, 4, 5, 0, 1, 2))` on the 6D matrix. What we want is symmetry in the 2D matrix sense, so everything is as it should be. Sorry for the confusion.

Reply:
I was able to move this away from here, but I will need to revisit `fb_3d` and `kernel.py`. I was hoping to have more success with it today, but its a bit of a rabbit hole. It will hold up forward progress.

Reply:
Ok let's leave it for now, then.